shell/platform/linux/fl_engine.cc

```diff
@@ -440,6 +464,10 @@ gboolean fl_engine_start(FlEngine* self, GError** error) {
   self->settings_plugin = fl_settings_plugin_new(self->binary_messenger);
   fl_settings_plugin_start(self->settings_plugin);
 
+  result = self->embedder_api.UpdateSemanticsEnabled(self->engine, TRUE);
```

Comment:
Is there a way to tell whether accessibility is being used at the system level? The mobile embeddings only enable this when an assistive technology is enabled at the OS level, so that we're not paying all of the overhead of maintaining the tree when it's not being used.

@cbracken Do you know how expensive this is for the engine to do? I don't know if it's something we need to worry about on desktop or not.

Reply:
There's a GSettings key (`gsettings get org.gnome.desktop.interface toolkit-accessibility`) that is set by gnome-settings-daemon when the screen-reader or the screen-keyboard is enabled. I haven't found anything that consumes this - GTK seems to generate all the ATK objects always regardless of them being consumed. This key may not be set on systems that aren't running gnome-settings-daemon and if we relied on it then a11y might not work on those systems.